### ISSUE TITLE
Add MYH7 Predictor to ClinGen Predictors table in Computational Tab

### ISSUE THREAD

**Issue by wrightmw** [ClinGen/clincoded#829]
This issue follows on from #772 where the other Bustamante predictors were added to the VCI. We  await further input from the MYH7 predictor creators before we can proceed with adding it to the ClinGen Predictors table. (also linked to #828, which could be merged into this ticket)
